### ISSUE TITLE
Add words and icon to practices to show which zone it sits in

### ISSUE THREAD

**Issue by kattekrab** [openpracticelibrary/openpracticelibrary#1466]
I've found myself getting confused about where a practice sits - the only indicator is the colour.

Could we/ should we add the tag as a word, and the icon to help show where a practice sits?

**Comment by rmarting** [comment]
This is a bit old issue, but the current status of the OPL site visualizes some icons and colors to identify the practice inside of the Mobius Loop. 

For example, the main pages has a navigation banner with icons per zone:

![image](https://github.com/user-attachments/assets/e511d8d1-eae5-496e-9b5f-4cc677a7fa17)

IMHO, this issue could be closed. Or are we missing something else?

**Comment by kattekrab** [comment]
@rmarting yes - the main page has the icons, but the practice pages themselves do not have the icons, or even any text suggesting which part of mobius they're in, other than the colour.

So let's say we focus on "Deliver"

![image](https://github.com/user-attachments/assets/d5c5dca3-b201-413d-a687-f53fedbbbc67)

and then click on [Autoscaling](https://openpracticelibrary.com/practice/autoscaling/)

![image](https://github.com/user-attachments/assets/f1b0c65b-306f-443f-97fe-32fcc2377d56)

We can see that this is Green and the same colour as the Deliver part of mobius, but there is no mobius shown on the practice page itself, and there is no matching icon, and it does not say "Deliver" anywhere either.

So, if you come to the Autoscaling page from an inbound link, we have no context for where it sits in mobius, or how it is categorised at all!

Does that make more sense now?

**Comment by rmarting** [comment]
@kattekrab Thanks for the explanation ... it helped me to propose an idea about where to present that information. The PR #2286 provides an approach about that design, of course, open to other options to visualize. I am not an UX designer, but I can implement other ideas successfully from that PR.

HTH

**Comment by kattekrab** [comment]
Amazing work @rmarting - thank you so much! I've just merged that in, and hoping it does not break anything, given the token issue, and tests not working. 🤞 

I have some suggestions for follow up tasks.

Instead of saying "A practice of $icon $mobiuspart" - perhaps just have $icon $mobiuspart
and perhaps that string could link to a filtered view of the home page?
So Foundation practices would link back to https://openpracticelibrary.com/tags/foundation/
and Decide practices would link back to https://openpracticelibrary.com/tags/options/
and so on for Discover and Deliver

what do you think @eliseelkerton and @rmarting

**Comment by kattekrab** [comment]
I've gone ahead and created a follow up issue, so we can close this one now. Thanks again @rmarting - I had wanted this tweak a loooong time!  and I love it.
